utils/page-utils.ts

```diff
@@ -75,12 +75,21 @@ export class PageUtils {
      *
      * @return {Promise<void>}
      */
-    public wpAdminLogin = async (): Promise<void> => {
+    public wpAdminLogin = async (user: string | null = null): Promise<void> => {
+
+        let username= WP_USERNAME;
+        let password= WP_PASSWORD;
+
+        if(user === 'admin2') {
+            username = WP_USERNAME2
+            password = WP_PASSWORD2
+        }
```

Comment:
```suggestion
        let username = user === 'admin2' ? WP_USERNAME2 : WP_USERNAME;
        let password = user === 'admin2' ? WP_PASSWORD2 : WP_PASSWORD;
```
make it simple and readable.